sql/_34_fig/cbrd_24563/cases/REGEXP_COUNT_unicode.sql

```diff
@@ -0,0 +1,18 @@
+-- CBRD-24563 : default regexp library change from cppstd to RE2
+set system parameters 'regexp_engine=re2';
+
+--REGEXP_COUNT
+SET NAMES utf8 COLLATE utf8_ko_cs;
+SELECT REGEXP_COUNT('가나다가나다가나다라', '(가나)다', 1);
+SELECT REGEXP_COUNT('가나 가나다라 마바사아 자차카타 파하', '[[:alpha:]]+', 1);
+SELECT REGEXP_COUNT('11억 8112만 5400 won 더하기 420만 4432 won', '[[:alpha:]]', 8);
+SELECT REGEXP_COUNT('Kłak Aleksander', '[[:alpha:]]+', 1);
```

Comment:
To test such a case that the character class doesn't match the Unicode character, The case is meaningful. Additionally, We should add test cases that have \u{hangul} pattern matching with the Unicode Korean characters.

Please refer to the Google RE2 syntax link that I wrote in the Acceptance Criteria section in JIRA.

Reply:
In RE2 manual Unicode pattern matching use '\u{Hangul}'. 
The suggested test case is added.